tools/README.md

```diff
@@ -196,12 +196,29 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     of a selection criterion. i.e Select all
                                     event logs except the ones which have
                                     application name as the input string.
-  -f, --filter-criteria  <arg>      Filter newest or oldest N eventlogs for
-                                    processing.eg: 100-newest (for processing
-                                    newest 100 event logs). eg: 100-oldest (for
-                                    processing oldest 100 event logs)
+  -f, --filter-criteria  <arg>      Filter newest or oldest N eventlogs based on filesystem
```

Comment:
lets change the order and put filesystem information last

Reply:
don't see that this got done?

Reply:
Sorry I had updated the order in this line but hadn't changed the order below in the examples section. Done now.